corehq/apps/export/forms.py

```diff
@@ -1076,3 +1077,85 @@ def extra_fields(self):
                 data_bind='value: dateRange',
             ),
         ]
+
+
+class IncrementalExportForm(forms.ModelForm):
+    # export_instance_id defined in __init__ to populate choices from request
+    active = forms.BooleanField(
+        label=_('Active'),
+        help_text=_('This export is enabled'),
+        required=False,
+        initial=True,
+    )
+
+    class Meta:
+        model = IncrementalExport
+        fields = [
+            'name',
+            'export_instance_id',
+            'connection_settings',
+            'active',
+        ]
+
+    def __init__(self, *args, request, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.domain = request.domain  # Passed by ``FormSet.form_kwargs``
+        self.fields['export_instance_id'] = forms.ChoiceField(
+            label=_('Case Data Export'),
+            choices=_get_case_data_export_choices(request),
+        )
+
+    def save(self, commit=True):
+        self.instance.domain = self.domain
+        return super().save(commit)
+
+
+class BaseIncrementalExportFormSet(forms.BaseModelFormSet):
+
+    def __init__(self, *args, request, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.form_kwargs['request'] = request  # Passed by ``FormView.get_form_kwargs()``
+
+
+IncrementalExportFormSet = forms.modelformset_factory(
+    model=IncrementalExport,
+    form=IncrementalExportForm,
+    formset=BaseIncrementalExportFormSet,
+    extra=1,
+    can_delete=True,
+)
+
+
+class IncrementalExportFormSetHelper(FormHelper):
```

Comment:
Could you inherit from `HQFormHelper`, or are you intentionally using slightly different label & field classes here?

Reply:
Not intentional at all. Thank you for the suggestion!